var/spack/repos/builtin/packages/py-tensorboard-data-server/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+
+from spack import *
+
+
+class PyTensorboardDataServer(PythonPackage):
+    """Fast data loading for TensorBoard"""
+
+    homepage = "https://github.com/tensorflow/tensorboard/tree/master/tensorboard/data/server"
+    git      = "https://github.com/tensorflow/tensorboard"
```

Comment:
Can we download from URL so that we can checksum? Or does tensorboard-data-server versioning not match up with tensorboard versioning?

Reply:
The data-server code is versioned differently and set at specific commits.